packages/itwin/ec3-widget/package.json

```diff
@@ -0,0 +1,114 @@
+{
+  "name": "@itwin/ec3-react",
+  "version": "1.0.0",
+  "description": "",
+  "main": "lib/cjs/e-c-3-widget.js",
```

Comment:
There should be no dashes in EC3. `"main": "lib/cjs/ec3-widget.js"

Reply:
Dashes have been removed